### ISSUE TITLE
Feature Request:

### ISSUE THREAD

**Issue by SchroederSteffen** [Autodesk/hubble#233]
Hi there,

We're using Hubble internally and it's a great tool, thanks!

One kind of KPI I'm missing is an aggregation over organizations for activity data.
On https://autodesk.github.io/hubble/orgs-activity, I would wish for a table that shows the most active organizations, just like it's done on https://autodesk.github.io/hubble/housekeeping-git-traffic for repositories.

Some orgs use mono-repos that win the repo-based table, but I would also like to see which orgs are the most active in general, over all repositories.
This could also be used for housekeeping purposes, e.g. if a single org has dozens of repos that bring down the system performance by using bad practices.

Cheers,
Steffen

**Comment by pramodhm112** [comment]
Hi @SchroederSteffen 

Please help me which version of GitHub Enterprise Server are you running

**Comment by SchroederSteffen** [comment]
Hi @pramodhm112,
We actually have two instances, one on `3.0.10` and one still on `2.22.10`.

**Comment by pramodhm112** [comment]
Is the Hubble working on both the instances and may I know about the configuration changes done if any please share it with us, bcs I'm getting error while I'm setting up this in our instance #231

**Comment by SchroederSteffen** [comment]
It's running on both instances, but we're using `3.0.10` while your issue seems to occur for `3.1.0`, no?
(Also, I'm not part of the infrastructure team that administrates it, so I can't help regarding the configuration changes, sorry.)

**Comment by pramodhm112** [comment]
Yes @SchroederSteffen for the confirmation. There's seems to be an issue in 3.0.8 also so I was checking in 

Thanks for Info, I guess we need to change the ghe-console commands since there are db changes in major version upgrades.
